OAuth/FirebaseOAuthUI/FUIOAuth.h

```diff
@@ -40,6 +40,7 @@ NS_ASSUME_NONNULL_BEGIN
     @param authUI The @c FUIAuth instance that manages controllers of this provider.
     @param providerID The unique identifier for the provider.
     @param buttonLabelText The text label for the sign in button.
+    @param shortName A short display name for the provider.
```

Comment:
for -> of?

Reply:
The name is pretty arbitrary. so I guess for is more appropriate?